### DISCUSSION TITLE
Generic lamp node: use variables to set values

### DISCUSSION THREAD

**Discussion by edufasolin** [Dirnei/node-red-contrib-zigbee2mqtt-devices#100] [Ideas]
In the generic-lamp node it would be interesting if we could set values such as brightness, temperature and colour using global/local variables, allowing us to use custom functions to determine these values.

How the node is:
![how_it_is](https://user-images.githubusercontent.com/80791051/111383368-00213e00-86a0-11eb-9e39-6f5b80c3ef5d.JPG)

How it is in Alexa Routine node, just as example (from https://flows.nodered.org/node/node-red-contrib-alexa-cakebaked)
![how_could_be](https://user-images.githubusercontent.com/80791051/111383043-a28cf180-869f-11eb-8baf-1c86e5da8583.JPG)

**Comment by Dirnei** [comment]
Thanks, that's an excellent idea! I wonder why I never thought of that myself...
